checks/fields.go

```diff
@@ -63,3 +63,12 @@ func CheckFieldRequiredness() *thriftcheck.Check {
 		}
 	})
 }
+
+// CheckFieldCommentMissing warns if a field doesn't have comments.
+func CheckFieldCommentMissing() *thriftcheck.Check {
```

Comment:
I think we can be a little more specific and refer to this as "documentation" or "a documentation comment" to avoid any ambiguity with "simple" comments.

Reply:
```suggestion
// CheckFieldDocMissing warns if a field doesn't have a documentation comment.
func CheckFieldDocMissing() *thriftcheck.Check {
```

Reply:
Thank you, I'll make the necessary changes.